tests/test_user_document.py

```diff
@@ -0,0 +1,25 @@
+"""Unit test module for terms of use logic"""
+import json
+from flask_webtest import SessionScope
+from tempfile import NamedTemporaryFile
+from StringIO import StringIO
+
+from tests import TestCase, TEST_USER_ID
+from portal.extensions import db
+from portal.models.user_document import UserDocument
+
+
+class TestUserDocument(TestCase):
+    """User Document tests"""
+
+    def test_post_patient_report(self):
+        #tests whether we can successfully post a patient report -type user doc file
+        service_user = self.add_service_user()
+        self.login(user_id=service_user.id)
+        tmpfile = NamedTemporaryFile(suffix='.pdf')
```

Comment:
I might add using `with` on resources ([see example](https://stackoverflow.com/questions/3924117/how-to-use-tempfile-namedtemporaryfile-in-python/21944302#21944302)) that need to be explicitly closed (like files) is a good habit to be in but not strictly necessary.